### ISSUE TITLE
Autoembedder breaks a lot of URLs

### ISSUE THREAD

**Issue by povrazor** [ludumdare/ludumdare#678]
Yeah, this is a problem. Autoembedder should only do its thing on URLs that you use without the markdown link tags. i.e.

```
# Like this
https://github.com/ludumdare

# Not like this
[my url](https://twitch.tv)
```

#676

**Comment by zwrawr** [comment]
You could still add the site image next to the name specified in the markdown.
You should also add a site image for ldjam.com and ludumdare.com

**Comment by povrazor** [comment]
Yeah, and itch.io, and gamejolt, etc. :wink:

But first it should be cleaned up.

I like the idea of dropping a little marker when a URL is to a known site (even if obscured by the `[]()` tags). I was considering that myself, but didn't want to complicate this. I'm just trying to get some of the super-broken problems of the site on the list.

**Comment by zwrawr** [comment]
The behaviour of the auto embedder fixed by #693

**Comment by povrazor** [comment]
I think for the most part this is fixed. URLs with hashes now work. New issues (such as the isInternal check) should be opened as a separate issue.
